app/uk/gov/hmrc/leakdetection/connectors/TeamsAndRepositoriesConnector.scala

```diff
@@ -73,4 +73,14 @@ object RepositoryInfo {
       implicit val hc = HeaderCarrier()
       http.GET[Option[RepositoryInfo]](url"${baseUrl}/api/v2/repositories/$repoName")
   }
+
+  def reposWithTeams(teamName: String): Future[Seq[RepositoryInfo]] = {
+      implicit val hc = HeaderCarrier()
+      http.GET[Seq[RepositoryInfo]](url"${baseUrl}/api/v2/repositories?team=$teamName")
+  }
+
+  def archivedRepos(): Future[Seq[RepositoryInfo]] = {
+      implicit val hc = HeaderCarrier()
```

Comment:
rather than creating a `HeaderCarrier` in each function on each call, it could be declared once in the Connector.